submissions/sprint_b/test_helpers.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+from helpers import FileManager, Vocabulary, EpithetGenerator
+
+test_path = ('/Users/ethansternke/Documents/Kenzie/back-end'
+             '/backend-epithet-generator/resources/test.json')
+
+test_vocab = (
```

Comment:
You're using this as an expected value in your assertions. You should fixtures
for this instead.
https://docs.pytest.org/en/latest/fixture.html#fixtures-as-function-arguments